packages/flutter/lib/src/widgets/parent_child_checkbox.dart

```diff
@@ -0,0 +1,183 @@
+import 'package:flutter/material.dart';
+
+///Main widget of Parent Child Checkbox.
+///Parent Child Checkbox is a type of checkbox where we can establish hierarchy in Checkboxes.
+class ParentChildCheckbox extends StatefulWidget {
+  ///Text Widget to specify the Parent checkbox
```

Comment:
docs should be full sentences and end with a `.`

Also, most docs in this file need improvements as they are not very informative (see also https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-useless-documentation).